cmd/bfgd/bfgd.go

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) 2024 Hemi Labs, Inc.
```

Comment:
You can't remove the copyright from an existing file.

Reply:
Ok